pkg/build/controller/factory/factory.go

```diff
@@ -115,6 +115,17 @@ func (factory *BuildControllerFactory) Create() controller.RunnableController {
 	}
 }
 
+func formatError(msg string) string {
```

Comment:
Put this in pkg/util/errors and make it ErrorToSentence and accept an err, not a string.